chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -74,16 +73,16 @@ public static DataSet averageDataSetsFIR(final List<DataSet> dataSets, final int
         if (dataSets.size() <= 1) {
             final DataSet newFunction = dataSets.get(0);
             if (newFunction instanceof DataSetError) {
-                return new DoubleErrorDataSet(functionName, values(DIM_X, newFunction), values(DIM_Y, newFunction),
+                return new DoubleErrorDataSet(functionName, newFunction.getValues(DIM_X), newFunction.getValues(DIM_Y),
                         errors(newFunction, EYN), errors(newFunction, EYP), newFunction.getDataCount(), true);
             }
 
             final int ncount = newFunction.getDataCount();
-            return new DoubleErrorDataSet(functionName, values(DIM_X, newFunction), values(DIM_Y, newFunction),
+            return new DoubleErrorDataSet(functionName, newFunction.getValues(DIM_X), newFunction.getValues(DIM_Y),
                     new double[ncount], new double[ncount], ncount, true);
         }
 
-        final int nAvg = Math.min(nUpdates, dataSets.size());
+        final int nAvg = java.lang.Math.min(nUpdates, dataSets.size());
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.min' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)